File: RxUI/src/main/java/com/tamsiree/rxui/view/cardstack/tools/RxScrollDelegate.java
package com.tamsiree.rxui.view.cardstack.tools;

/**
 * @author tamsiree
 * @date 2018/6/11 11:36:40 整合修改
 */
public interface RxScrollDelegate {

    void scrollViewTo(int x, int y);

    int getViewScrollY();

    void setViewScrollY(int y);

    int getViewScrollX();

    void setViewScrollX(int x);

}

File: RxUI/src/main/java/com/tamsiree/rxui/view/cardstack/tools/RxScrollDelegate.kt
package com.tamsiree.rxui.view.cardstack.tools

/**
 * @author tamsiree
 * @date 2018/6/11 11:36:40 整合修改
 */
interface RxScrollDelegate {
    fun scrollViewTo(x: Int, y: Int)
    var viewScrollY: Int
    var viewScrollX: Int
}